pages/SubmitForm/index.js

```diff
@@ -0,0 +1,11 @@
+import Form from "../../components/Form/index.js";
+import { useRouter } from "next/router";
+
+export default function SubmitForm(locationData) {
+  const router = useRouter();
+
+  const handleFormSubmit = (formData) => {
+    router.push("/LocationSuccess");
+  };
```

Comment:
You're not using `formData`. Remove it if you don't need it or make use of it of you do need it.